components/popup/index.tsx

```diff
@@ -20,14 +20,16 @@ const Popup: React.FC<{
     <>
       <Backdrop onClick={onClose} />
       {createPortal(
-        <article
+        <div
```

Comment:
This one was a little tricky 😄. All page content must be contained in landmarks. When the dialog is showing, it needs something like `role="dialog"` to indicate it's a dialog landmark.

https://dequeuniversity.com/rules/axe/4.7/region?application=AxeChrome